docs/advanced_reference/commands/autonomy_mint.md

```diff
@@ -0,0 +1,126 @@
+# `autonomy mint`
```

Comment:
Do not use top-level titles. These are included in mkdocs.yaml. See other command group (e.g., autonomy build) to see how it is structured.